core/src/main/java/io/questdb/griffin/engine/functions/catalogue/ShowServerVersionNumCursorFactory.java

```diff
@@ -0,0 +1,119 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2024 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.griffin.engine.functions.catalogue;
+
+import io.questdb.cairo.AbstractRecordCursorFactory;
+import io.questdb.cairo.ColumnType;
+import io.questdb.cairo.GenericRecordMetadata;
+import io.questdb.cairo.TableColumnMetadata;
+import io.questdb.cairo.sql.Record;
+import io.questdb.cairo.sql.RecordCursor;
+import io.questdb.griffin.PlanSink;
+import io.questdb.griffin.SqlExecutionContext;
+
+public class ShowServerVersionNumCursorFactory extends AbstractRecordCursorFactory {
+    private static final GenericRecordMetadata METADATA = new GenericRecordMetadata();
+    private static final int SIZE = 1;
+    private final ShowServerVersionNumRecordCursor cursor = new ShowServerVersionNumRecordCursor();
+
+    public ShowServerVersionNumCursorFactory() {
+        super(METADATA);
+    }
+
+    @Override
+    public RecordCursor getCursor(SqlExecutionContext executionContext) {
+        return cursor;
+    }
+
+    @Override
+    public boolean recordCursorSupportsRandomAccess() {
+        return false;
+    }
+
+    @Override
+    public void toPlan(PlanSink sink) {
+        sink.type("show_server_version_num");
+    }
+
+    private static class ShowServerVersionNumRecordCursor implements RecordCursor {
```

Comment:
nit: should extend `NoRandomAccessRecordCursor` to avoid stubbing

Reply:
Done in 0e0cd23673d2f5fc2ac63119be1b8cfc958d5d89